airflow/api_fastapi/core_api/routes/public/dag_run.py

```diff
@@ -329,7 +329,11 @@ def get_dag_runs(
     ),
 )
 def trigger_dag_run(
-    dag_id, body: TriggerDAGRunPostBody, request: Request, session: SessionDep
+    dag_id,
+    body: TriggerDAGRunPostBody,
+    request: Request,
+    session: SessionDep,
+    _: None = Depends(action_logging("trigger_dag_run")),
```

Comment:
Depends that are not needed can go directly into the `dependencies` of the route decorator.

Reply:
Also we need to add back the logging for all endpoints, based on the legacy api action logging.

(Basically taking all the endpoint 1 by 1 and adding the appropriate action logger on each of them).